jsapp/js/components/common/koboDropdown.tsx

```diff
@@ -257,7 +257,7 @@ export default class KoboDropdown extends React.Component<
         {this.state.isMenuVisible && (
           <bem.KoboDropdown__menu
             onClick={this.onMenuClick.bind(this)}
-            aria-role='listbox'
+            role='listbox'
```

Comment:
unrelated to colors, but this makes a console warning go away: an aria prop warning in the console in dev mode since [`8ba6ce7e5`](https://github.com/kobotoolbox/kpi/commit/8ba6ce7e5fc0b0a02a6c8251d05b7290884daf56).